components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java

```diff
@@ -2448,7 +2448,8 @@ private boolean validateUpdateOfIdPEntityId(FederatedAuthenticatorConfig[] curre
 
     private String getOIDCResidentIdPEntityId() {
 
-        String OIDCEntityId = IdentityUtil.getProperty("OAuth.OpenIDConnect.IDTokenIssuerID");
+        String OIDCEntityId = IdentityUtil.resolveURL(IdentityUtil.getProperty("OAuth.OpenIDConnect.IDTokenIssuerID")
```

Comment:
Is this method only used during resident idp creation?

Reply:
yes.

Reply:
Its used in the getResidentIdp path as well. But the method seems to server as a way to get the default value of the IDTokenIssuer so fine to include to logic to tenant qualify the value.